packages/core/src/codewhisperer/activation.ts

```diff
@@ -269,6 +267,11 @@ export async function activate(context: ExtContext): Promise<void> {
         )
     )
 
+    const connectWithCustomizationCommand = connectWithCustomization()
+    if (connectWithCustomizationCommand) {
+        context.extensionContext.subscriptions.push(connectWithCustomizationCommand.register())
+    }
```

Comment:
can avoid this with https://github.com/aws/aws-toolkit-vscode/pull/4631/files#r1552588811